rpcs3/Emu/Cell/lv2/sys_memory.cpp

```diff
@@ -1,4 +1,4 @@
-#include "stdafx.h"
+﻿#include "stdafx.h"
```

Comment:
Why is BOM necessary here?

Reply:
This project uses BOM for some reason
https://github.com/RPCS3/rpcs3/blob/0e0a82e5363cabdfd417d907847232ececb4fdd5/.editorconfig#L4
It's stupid, but that's what the project has decided to do.

Reply:
This was decided after multiple contributors had issues where an editor or the compiler didn't auto-detect UTF-8 properly and broke stuff such as log messages.